Ebury/chameleon
# Updated storybook to v5.2.6 and added story sorting.

To verify:
1. Run storybook and check all addons are working, the theming is still there.
2. Verify the stories are sorted alphabetically and they stay sorted after HMR reloads any story.
